README.md

```diff
@@ -59,8 +85,9 @@ Creating a task manifest file.
     "version": 1,
     "schedule": {
         "type": "simple",
-        "interval": "1s"
+        "interval": "100s"
```

Comment:
Sliding factor is 1 by default. So it's the interval.

Maybe 100 seconds is a bit too much for a task example?